homework-submission/week1-homework/README.md

```diff
@@ -0,0 +1,3 @@
+Yo can reach the week1 Homework from the link below:
```

Comment:
Please only add the files of the week3 to your PR!

Reply:
Thanks Paul it seems likely I forgot to checkout branch master, sorry.